src/panels/lovelace/cards/hui-map-card.js

```diff
@@ -0,0 +1,258 @@
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+import Leaflet from 'leaflet';
+
+import '../../map/ha-entity-marker.js';
+
+import processConfigEntities from '../common/process-config-entities';
+import computeStateDomain from '../../../common/entity/compute_state_domain.js';
+import computeStateName from '../../../common/entity/compute_state_name.js';
+
+Leaflet.Icon.Default.imagePath = '/static/images/leaflet';
+
+class HuiMapCard extends PolymerElement {
+  static get template() {
+    return html`
+      <style>
+        ha-card:not(.panel) {
+          line-height: 0;
```

Comment:
Why?

Reply:
This actually isn't needed, it got left from when I was copying the aspect_ratio stuff from iframe. WIll remove.